internal/controller/operatorconfigmap_controller.go

```diff
@@ -342,6 +356,17 @@ func (c *OperatorConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			return ctrl.Result{}, err
 		}
 
+		autoApproveInstallPlans, err := strconv.ParseBool(cmp.Or(c.operatorConfigMap.Data[autoApproveInstallPlansKey], "true"))
```

Comment:
1. by having a key where the value need to be true in order to exec a code path it automaticaly implies that if it does not exists the code path should not be executed. I am not talking about the code being wrong I am talking about API design principles 

2. This code already have an issue as it defaults to false if the value is not proper bool which differs to defulating to true if the value does not exists.

My request is to flip the logic so the key will indicate that we are to "disable" auto approval, it will solve both problems in one go

```suggestion
		disableInstallPlanAutoApproval, err := strconv.ParseBool(c.operatorConfigMap.Data[DisableInstallPlanAutoApproval])
```